src/bull.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const { randomNumber } = require('./getRandomNumber');
+
+function bull(userInput) {
+  let count = 0;
+
+  const arrInput = userInput.split('');
+  const arrNumber = randomNumber.split('');
+
+  for (let i = 0; i < arrNumber.length; ++i) {
```

Comment:
```suggestion
  for (let i = 0; i < arrNumber.length; i++) {
```
It's safer to use postfix increment. It always returns incremented value, prefix increments laso but returns previous value. Please, replace it all over your code.

Reply:
By the way, about the prefix increment, I can say that it is safe to use this thing in loops, since this operation is performed at the end of the iteration, and with a prefix increment, a copy of the variable is not created to add a value to it, how it is done with postfix